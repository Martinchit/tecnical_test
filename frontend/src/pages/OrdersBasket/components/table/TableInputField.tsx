import styled from 'styled-components';
import { FormControl } from 'react-bootstrap';

export const TableInputField = styled(FormControl)`
  width: 100%;
`;
