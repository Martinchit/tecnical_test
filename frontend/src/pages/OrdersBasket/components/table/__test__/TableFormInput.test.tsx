import React from 'react';
import { shallow } from 'enzyme';

import { TableInputField } from '../TableInputField';

describe('TableInputField', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<TableInputField />);
    expect(wrapper).toMatchSnapshot();
  });
});
