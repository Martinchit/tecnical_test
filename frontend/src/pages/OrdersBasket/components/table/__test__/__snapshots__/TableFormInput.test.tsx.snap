// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableInputField renders correctly 1`] = `
<FormControl
  className="sc-AxjAm jdsAYC"
/>
`;
